docs/team/cheeerynne.md

```diff
@@ -11,62 +11,64 @@ that are typed into the input box as commands and it has a GUI created with Java
     * Highlights: The implementation was challenging as I had to check whether the specified member's name exists in the 
       members' list before allowing assignment to a task. I decided to implement an Assignee class, along with a checkAssignees 
-      method within the ModelManager class. The checkAssignees method checks the validity of all the specified member's name by 
-      calling the hasPerson method within the Model class while not creating any unnecessary dependency between the Task and Person class.
-      Furthermore, integration and unit testing had to be implemented thoroughly to ensure that the various scenarios of usage were accounted for.
+      method within the ModelManager class. The checkAssignees method checks the validity of the specified member's name by 
+      calling the hasPerson method within the ModelManager class. With the checkAssignees method, I was able to check 
+      the validity of the specified names while not creating any unnecessary dependency between the Task and Person class.
 
-**New Feature**: Added the ability to find tasks with specified keywords. (Pull Requests: [\#133](https://github.com/AY2021S2-CS2103T-W14-3/tp/pull/133))
-    * What it does: allows the user to find all tasks which contains any of the specified keywords in its title or description. 
+* **New Feature**: Added the ability to find tasks with specified keywords. (Pull Requests: [\#133](https://github.com/AY2021S2-CS2103T-W14-3/tp/pull/133))
+    * What it does: allows the user to find all tasks which contain any of the specified keywords in its title or description. 
     * Justification: This feature improves the product significantly because a user can easily filter and find tasks based on specific 
       keywords. With this easy filtering, there is no need for users to scroll through the entire list just to find a specific task. 
-    * Highlights: The implementation was challenging as I had to check only the title and description fields of a task. 
-      I decided to implement a predicate, which takes in the title and description of a task, to check if any of these 2 fields
-      matches any of the keywords given. Furthermore, integration and unit testing had to be implemented thoroughly to ensure 
-      that the various scenarios of usage were accounted for.
-
-**New Feature**: Added the ability to find members with specified keywords. (Pull Requests: [\#130](https://github.com/AY2021S2-CS2103T-W14-3/tp/pull/130))
-    * What it does: allows the user to find all members whose name, phone number, email or role any of the specified keywords.
+    * Highlights: The implementation was challenging as I had to check through the title and description field of all the tasks 
+      in the task list. I decided to implement a predicate,TaskContainsKeywordPredicate, which compares the title and description of
+      a task to the list of keywords specified. The task list is then updated accordingly to the predicate. 
```

Comment:
Should refer to the filtered task list for more clarity

Reply:
findTasks does not check through the filtered task list, it checks through the unfiltered task list.